activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+*   Allow ActiveRecord::QueryMethods#reselect to receive hash values, similar to ActiveRecord::QueryMethods#select
+
+    *Sampat Badhe*
```

Comment:
You are missing a space here.

Reply:
space -> blank line, sorry.

Reply:
Thank you for the review @byroot I made the requested change.